dash/webapp/pages/pa11y.tsx

```diff
@@ -193,6 +187,31 @@ const Index: FunctionComponent = () => {
                   );
                 }
               )}
+
+              {successes.length > 0 && (
+                <section
+                  key="successes"
+                  style={{
+                    marginTop: '18px',
+                    padding: '6px 0',
+                    borderTop: '1px solid #d9d6ce',
+                  }}
+                >
+                  <Issue type="success">
+                    No issues reported on the following pages.
+                  </Issue>
+
+                  <ul>
+                    {successes.map(({ pageUrl, documentTitle }) => (
+                      <li key={pageUrl}>
+                        <OriginalPageLink href={pageUrl}>
+                          {documentTitle.replace('| Wellcome Collection', '')}
```

Comment:
This `.replace()` is because our page titles follow a standard formula; repeating it is just noise on the page.